auth/postgres/groups.go

```diff
@@ -596,26 +631,32 @@ func toGroup(dbu dbGroup) (groups.Group, error) {
 }
 
 type dbGroupRelation struct {
-	GroupID  uuid.UUID `db:"group_id"`
-	MemberID uuid.UUID `db:"member_id"`
+	GroupID   sql.NullString `db:"group_id"`
+	MemberID  sql.NullString `db:"member_id"`
+	CreatedAt time.Time      `db:"created_at"`
+	UpdatedAt time.Time      `db:"updated_at"`
+	Type      string         `db:"type"`
 }
 
-func toDBGroupRelation(memberID, groupID string) (dbGroupRelation, error) {
-	grID, err := uuid.FromString(groupID)
-	if err != nil {
-		return dbGroupRelation{}, err
+func (gr groupRepository) toDBGroupRelation(memberID, groupID, groupType string) (dbGroupRelation, error) {
+	var grID sql.NullString
+	if groupID != "" {
+		grID = sql.NullString{String: groupID, Valid: true}
```

Comment:
Why recreate a `NullString` instance when you can assign values to it's members.

Reply:
null value and empty string are not the same thing in the db